packages/core/src/extension/extension.ts

```diff
@@ -3,6 +3,7 @@ export interface Extension {
     name: string;
     region: ExtensionRegion;
     url: string;
+    type?: 'iframe' | 'worker';
```

Comment:
Can we type this as an enum, and then we can use the value here https://github.com/bigcommerce/checkout-sdk-js/pull/2869/files